middlewares/validators/task-requests.js

```diff
@@ -1,8 +1,10 @@
+import { GITHUB_URL } from "../../constants/urls";
 const joi = require("joi");
 const { RQLQueryParser } = require("../../utils/RQLParser");
 const githubOrg = config.get("githubApi.org");
 const githubBaseUrl = config.get("githubApi.baseUrl");
 const githubIssuerUrlPattern = new RegExp(`^${githubBaseUrl}/repos/${githubOrg}/.+/issues/\\d+$`);
+const githubIssueHtmlUrlPattern = new RegExp(`^${GITHUB_URL}/${githubOrg}/.+/issues/\\d+$`);
```

Comment:
Please add what type of pattern are you checking against with a example in description

Reply:
added